test_runner/fixtures/pageserver/allowed_errors.py

```diff
@@ -112,6 +112,9 @@ def scan_pageserver_log_for_errors(
     # controller's attempts to notify the endpoint).
     ".*reconciler.*neon_local notification hook failed.*",
     ".*reconciler.*neon_local error.*",
+    # Neon local does not provide pageserver with an AZ
```

Comment:
Is it non-trivial to add a placeholder ID in neon_local in this PR, so that we don't have to remember this TODO later?

Reply:
Done in the next [PR](https://github.com/neondatabase/neon/pull/8856).